src/tokens/evo-dark.css

```diff
@@ -40,7 +44,9 @@
         --color-state-primary-hover: #17212c;
         --color-state-primary-active: #1e2a38;
         --color-state-secondary-hover: #3e3e3e;
+        --color-state-secondary-hover-rgb: 62, 62, 62;
```

Comment:
Just curious, why are these rgb colors needed?

Reply:
For the overflow it has a background color and is semi transparent. In order to use apply both the background color and alpha at the same time, I'm using `rgba` and it only takes `rgb` instead of hex. This is similar to what was done in this PR for table 

https://github.com/eBay/skin/pull/2423/files#diff-17bb714f451f6f33c164d2bcecac7b9cfa6a2baffa5490ca6941e8747d1ce358R144